constraints.txt

```diff
@@ -1,3 +1,3 @@
 sphinxcontrib-serializinghtml===1.1.5
 # since version 7.2.0 the introduced changes in sphinx.builders.html (function setup_resource_paths) cause the json jobs fail
-sphinx===7.1.2
+sphinx>=7.1.4
```

Comment:
should be `sphinx>=7.4` see https://www.sphinx-doc.org/en/master/latex.html
![grafik](https://github.com/user-attachments/assets/cdadde1e-44e7-4633-9081-9045e8cb9f17)